packages/medusa/src/api/routes/admin/discounts/create-condition.ts

```diff
@@ -0,0 +1,104 @@
+import { IsOptional, IsString } from "class-validator"
+import { defaultAdminDiscountsFields, defaultAdminDiscountsRelations } from "."
+import { Discount } from "../../../../models/discount"
+import { DiscountConditionOperator } from "../../../../models/discount-condition"
+import { DiscountService } from "../../../../services"
+import DiscountConditionService from "../../../../services/discount-condition"
+import { AdminUpsertConditionsReq } from "../../../../types/discount"
+import { getRetrieveConfig } from "../../../../utils/get-query-config"
+import { validator } from "../../../../utils/validator"
+/**
+ * @oas [post] /discounts/{discount_id}/conditions
+ * operationId: "PostDiscountsConditions"
```

Comment:
todo: PostDiscountsDiscountConditions
```suggestion
 * operationId: "PostDiscountsDiscountConditions"
```